label_sync/main.go

```diff
@@ -777,7 +777,9 @@ func main() {
 		if deprecated {
 			githubClient, err = newClient(o.token, o.tokens, o.tokenBurst, !o.confirm, o.graphqlEndpoint, o.endpoint.Strings()...)
 		} else {
-			githubClient, err = o.github.GitHubClient(!o.confirm)
+			if err = o.github.Validate(!o.confirm); err == nil {
```

Comment:
Can we split this onto two lines? While I know what you have works (because you aren't `:=` reassigning err which would scope it to the if statement), I think splitting this onto a couple lines makes it more clear and specific that the err is being checked on L785.
```suggestion
			err = o.github.Validate(!o.confirm)
			if err == nil {
```